### DISCUSSION TITLE
CSS compatibility

### DISCUSSION THREAD

**Discussion by ZeroIntensity** [Textualize/textual#355] [Q&A] [answered]
i realize that terminal css is still in development, but how much of the actual css syntax will be implemented? the whole standard or just the basics? if it's the whole thing, then using css frameworks inside textual could be possible. would be really interesting to see textual alongside something like postcss

**Comment by USLTD** [comment, marked as answer]
It will support only basics and few advanced things since Terminal is not Web Browser or even Graphical application

**Comment by ThatXliner** [comment]
I think it would be nice if we didn’t need processors and instead have, say, SCSS built-in
